proto/rill/runtime/v1/queries.proto

```diff
@@ -323,9 +323,11 @@ message MetricsViewAggregationRequest {
   // Required
   repeated MetricsViewAggregationMeasure measures = 4;
   // Optional. Defaults to unsorted
-  repeated MetricsViewAggregationSort sort = 5;
+  repeated MetricsViewComparisonSort sort = 5;
```

Comment:
Is there a reason you didn't add configuration for the comparison in the `MetricsViewAggregationMeasure` type as discussed here? https://www.notion.so/rilldata/Pivot-APIs-improvements-8ee3db49882e4093af85de68198f5984

What should the frontend put in `measures` to request a comparison value?

Also, if keeping this, we should rename the type to `MetricsViewAggregationSort` to align with the API.

Reply:
+1 to this. There is no way to add having filter on comparison otherwise. So might need to build and pass `aliases` to `buildExpression` for having clause.

Reply:
yes, it's the next step to adapt it to the new proposal